Firmware/mmu2.cpp

```diff
@@ -313,15 +316,14 @@ bool MMU2::VerifyFilamentEnteredPTFE()
     if (!fsensor.getFilamentPresent()) return false;
 
     uint8_t fsensorState = 0;
-    // MMU has finished its load, move filament back by ExtraLoadDistance
-    // If Fsensor reads 0 at any moment, then report FAILURE
-    current_position[E_AXIS] -= logic.ExtraLoadDistance();
+    // MMU has finished its load, push the filament further by some defined constant length
+    // If the filament sensor reads 0 at any moment, then report FAILURE
+    current_position[E_AXIS] += MMU2_EXTRUDER_PTFE_LENGTH + MMU2_EXTRUDER_HEATBREAK_LENGTH;
```

Comment:
This can't be 60 anymore, you need to subtract the ExtraLoadDistance from this value or you'll make blobs on the tower.